Routers/BookingRouter.cs

```diff
@@ -0,0 +1,118 @@
+﻿using AirportTicketBooking.Classes;
+using AirportTicketBooking.Repositories;
+using ClassType = TicketClassType.ClassType;
+
+namespace AirportTicketBooking.Routers
+{
+    internal class BookingRouter
+    {
+        private BookingRepositoty _bookingRepositoty;
+        private FlightRouter _flightRouter;
+
+        public BookingRouter()
+        {
+            _bookingRepositoty = new BookingRepositoty();
+        }
+
+        public List<Booking> GetAllData()
+        {
+            return _bookingRepositoty.Load();
+        }
+
+        public List<Booking> GetAll()
+        {
+            return GetAllData().Where(booking => booking.ValidationMessage == null).ToList();
```

Comment:
```suggestion
            return GetAllData().Where(booking => string.IsNullOrEmpty(booking.ValidationMessage)).ToList();
```
since it's not nullable?